README.md

```diff
@@ -1 +1,25 @@
-# hw01
\ No newline at end of file
+# hw01
+
+## A Little bit about me.
+
+Hello my name is Arturo Juarez. I am studying **International Relations** under the Committee of International Relations at the *University of Chicago*. I was born in California but I grew up in both Illinios and and Southern California. As such I consider myself both a Midwesterner and a Californian. My interests incluce reading, outdoor running, and traveling to new destinations both domestically and internationally. 
+
+### More about me
+
+My *hobbies* include playing sports such as basketball, baseball, and volleyball. I enjoy the **outdoors** as much as I enjoy spending a lazy day **indoors**. I care deeply about the social, economic, and civil challenges facing our democracy today and I actively donate my time, energy, and resources to many causes I find of pressing importance. I *source* my news at times from [link](https://www.npr.org/) [link](https://www.washingtonpost.com/) or [link](https://thehill.com/) 
+
+#### Lists
+
+* Unordered List
```

Comment:
Unfortunately, neither of your lists rendered correctly. I can tell that you knew what syntax you needed to create an unordered and ordered list, but I think the execution just wasn't quite there

Reply:
Additionally, I would instead make "Unordered List" a section header and start the unordered list under it, or even use the list as an opportunity to tell us more about yourself